### PR TITLE
Fix bot token expiry docs links

### PR DESCRIPTION
**Pull request by sadohert**
Fixed the API link we use, and added some additional bullets on how the flow for refreshing a bot token would work.

### CODE REVIEW

**Comment by sadohert** [comment]
@cwarnermm @justinegeffen - I can't seem to add reviewers to the issue.  Maybe I need to comment to you first?

**Comment by cwarnermm** [comment]
Thanks for the heads-up, @sadohert! @cpanato - Can you review @sadohert's docs repo permissions, please? He's had challenges contributing to the docs repo, and it would be great to reduce that friction. We love @sadohert's contributions :)

**Comment by cwarnermm** [comment]
@sadohert - The developer-docs repo requires a second approver. Who would you recommend for a technical review of this content?

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:145]:
```suggestion
No, but you can automate your integration to cycle its token through the REST API:
```

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:146]:
```suggestion
1. Use [CreateUserAccessToken](https://api.mattermost.com/#operation/CreateUserAccessToken) to generate a new token.
```

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:147]:
```suggestion
2. Update the services that leverage the token with the new `token` value from step 1.
```

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:148]:
```suggestion
3. Use RevokeUserAccessToken](https://api.mattermost.com/#operation/RevokeUserAccessToken) to revoke the old token based on the `token_id`.
```

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

**Comment by cpanato** [comment]
> Thanks for the heads-up, @sadohert! @cpanato - Can you review @sadohert's developer-docs repo permissions, please? He's had challenges contributing to the docs repo, and it would be great to reduce that friction. We love @sadohert's contributions :)

should be good now

**Comment by sadohert** [comment]
@cwarnermm  - I'm not sure actually.  @amyblais or @jasonblais - Is there someone in engineering familiar with the bot stuff that you can recommend?

_btw, my permissions are updated.  i can add reviewers.  Thanks Carlos!_

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (isacikgoz) [inline site/content/integrate/admin-guide/admin-bot-accounts/_index.md:148]: We can add another bullet here to inform about listing tokens from https://api.mattermost.com/#operation/GetUserAccessTokensForUser so that they can find the token IDs.

**Review by isacikgoz — COMMENTED** [review]
Looks good, nice catch @sadohert 💪 Just have a comment below.

**Review by isacikgoz — APPROVED** [review]
Approving to unblock, I'm 0/5 on my suggestion above.

**Review by hanzei — COMMENTED** [review]
:+1:

File: site/content/integrate/admin-guide/admin-bot-accounts/_index.md
Comment (sadohert) [reply site/content/integrate/admin-guide/admin-bot-accounts/_index.md]: Additional bullet added.  Thanks @isacikgoz

**Review by cwarnermm — APPROVED** [review]
Thank you for this great developer documentation update!

**Pull request merged**
